api/src/main/java/org/openmrs/module/systemmonitor/export/DHISGenerateDataValueSetSchemas.java

```diff
@@ -174,6 +176,10 @@ public static JSONObject generateRwandaSPHEMTDHISDataValueSets() {
 					+ DHISMapping.getDHISMappedObjectValue("DATA-ELEMENT_totalEncounters") + "\", \"period\": \""
 					+ dFormat.format(today) + "\", \"orgUnit\": \"" + dhisOrganizationUnitUid + "\", \"value\": "
 					+ encounterTotal + "}";
+			String newEncountersDataElement = "{ \"dataElement\": \""
```

Comment:
we probably need to pass/add this to a json parser and add its json to jsonDataValueSets before its tapped into jsonToBePushed